src/Symfony/Bundle/FrameworkBundle/Resources/config/web.xml

```diff
@@ -70,5 +70,10 @@
         <service id="validate_request_listener" class="Symfony\Component\HttpKernel\EventListener\ValidateRequestListener" public="true">
             <tag name="kernel.event_subscriber" />
         </service>
+
+        <service id="resolve_controller_name_subscriber" class="Symfony\Bundle\FrameworkBundle\EventListener\ResolveControllerNameSubscriber" public="true">
```

Comment:
should not be public. Listeners can be private now, and there is no BC issue about making this one private as it is new